python/paddle/fluid/layers/nn.py

```diff
@@ -8472,3 +8473,74 @@ def bilinear_tensor_product(x,
 
     # add activation
     return helper.append_activation(out)
+
+
+@templatedoc()
+def tree_conv(nodes_vector,
+              edge_set,
+              output_size,
+              name=None,
+              num_filters=1,
+              max_depth=2,
+              act='tanh',
+              param_attr=None,
+              bias_attr=None):
+    """
+    ${comment}
+
+    Args:
+        nodes_vector(${nodes_vector_type}): ${nodes_vector_comment}
+        edge_set(${edge_set_type}): ${edge_set_comment}
+        output_size(int): output feature width
+        num_filters(int): number of filters, default 1
+        max_depth(int): max depth of filters, default 2
+        act(str): activation function, default tanh
+        param_attr(ParamAttr): the parameter attribute for the filters, default None
+        bias_attr(ParamAttr): the parameter attribute for the bias of this layer, default None
+
+    Returns:
+        out(${out_type}): ${out_comment}
+
+    Examples:
+        .. code-block:: python
+          nodes_vector = layers.data(name='vectors', shape=[None, 10, 5], dtype='float32)
+          # nodes_vector tensor
```

Comment:
No need for `#`?

Reply:
I'll fix my code with new Memory mechanism and commit later, and I'll remove the comment sign .

Reply:
please add a blank line between line8505-8506 to polish annotation correctly. @shanyi15 could you check the annotation of this API?